NOSQL/Assignment_doument_structure.txt

```diff
@@ -0,0 +1,37 @@
+Users
+{
+	id:1
+	user_name:"abc",
+	password:"zbxc123"
+	name:"anc asd",
+	date_of_birth:12-12-1990,
+	gender:"m or f"
+}
+
+Posts
+{
+	author_id:1
+	title:"post_title",
+	date_of_post:"11-2-2016 10:30",
+	content:{images,etc},
+	tags:{}
+	comments:[
+			{	
+				time_of_comment:"12-3-2016 6:10",
+				commentor:"asf",
+				comment:"hey1"
+			},
+			{	
+				time_of_comment:"12-4-2016 8:10",
+				commentor:"afg",
+				comment:"hey2"
+			}
+		 ]
+}
+Types_of_users
```

Comment:
This is incorrect,
- If it is a 1:1 relationship with users, you don't need a separate collection for this . 
- Storing multiple keys for each user denotes that the user can have multiple roles, which isn't correct as per the problem statement provided. 
- What about the users /posts assigned to a particular role 
eg: An editor can access only those posts assigned to him/her. Similarly admin's can access only those users, and posts of those users assigned to him/her

REWORK on this.